docs/src/pages/components/data-grid/editing/StartEditButtonGrid.tsx

```diff
@@ -100,11 +100,11 @@ export default function StartEditButtonGrid() {
     [],
   );
 
-  // Prevent from committing on blur
-  const handleCellBlur = React.useCallback(
-    (params, event?: React.SyntheticEvent) => {
-      if (params.cellMode === 'edit') {
-        event?.stopPropagation();
+  // Prevent from committing on focus out
+  const handleCellFocusOut = React.useCallback(
+    (params, event?: MouseEvent | React.SyntheticEvent) => {
+      if (params.cellMode === 'edit' && event) {
+        (event as any).defaultMuiPrevented = true;
```

Comment:
you could do an interface to make this a bit better
```tsx
interface MuiEvent extends React.SyntheticEvent {
  defaultMuiPrevented?: boolean
}
``` 
Also when you publish events you could use `as MuiEvent`